src/reanimated2/hook/useAnimatedSensor.ts

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useRef } from 'react';
+import { makeMutable } from '../core';
+import NativeReanimated from '../NativeReanimated';
+import { SensorValue3D, SensorValueRotation } from '../commonTypes';
+
+export enum SensorType {
+  ACCELEROMETER = 1,
+  GYROSCOPE = 2,
+  GRAVITY = 3,
+  MAGNETIC = 4,
```

Comment:
```suggestion
  MAGNETIC_FIELD = 4,
```
BTW why do we have to define this enum also in `react-native-reanimated.d.ts`?

Reply:
I didn't find a way to import enum to `d.ts` :/